can-route-pushstate.md

```diff
@@ -8,40 +8,61 @@
 
```

Comment:
The description should say something about why someone would use it.  This changes `can-route` from updating the hash to updating the url with either `pushState` or `replaceState`.